lms/envs/common.py

```diff
@@ -1291,6 +1295,18 @@
         ],
         'output_filename': 'css/lms-style-course-vendor.css',
     },
+    'style-edx-footer': {
```

Comment:
@AlasdairSwan, also heads up that I have this Post IE9-world PR that will shortly be merged - https://github.com/edx/edx-platform/pull/8035. You may want to sync up your css and css group names based on that.